pom.xml

```diff
@@ -119,6 +119,9 @@
     <zeppelin.version>0.9.0-SNAPSHOT</zeppelin.version>
     <jgit.version>5.5.1.201910021850-r</jgit.version>
     <atomix.version>3.0.0-rc4</atomix.version>
+    <libthrift.version>0.12.0</libthrift.version>
```

Comment:
Why need add `libthrift.version` ?

Reply:
Thanks to point it , the libthrift dependency is not needed for this PR, I will remove it.